website/docs/r/opa_version.html.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_opa_version"
+description: |-
+  Manages OPA versions
+---
+
+# tfe_sentinel_version
+
+Manage OPA versions available on Terraform Cloud/Enterprise.
```

Comment:
The intro to both of these doc pages should call out the admin role requirement, given that BOTH the author of the PR AND the reviewer stubbed their toes on that!! 🤣 Probably the terraform version page needs it as well, if you've got a sec, but I won't require that. 

```suggestion
Manage OPA versions available on Terraform Enterprise.

~> **Important:** This resource is only available in Terraform Enterprise, and the user account you use to manage the resource must have the "Version Maintenance" admin settings role. (Note that this role is mutually exclusive with the interactive "Site Admin" role.)
```